### PR TITLE
RELATED: ONE-4642 Move Timepicker from goodstrap to sdk-ui-kit

### PR DESCRIPTION
**Pull request by NestorEncinas**
---

Supported PR commands:

| Command         | Description            |
| --------------- | ---------------------- |
| `ok to test`    | Re-run standard checks |
| `extended test` | BackstopJS tests       |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [x] `check` passes
-   [x] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1133/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/2738/ : SUCCESS in 35m 28s

**Comment by NestorEncinas** [comment]
extended test

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1134/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/395/ : UNSTABLE in 17m 45s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1135/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/2739/ : SUCCESS in 15m 42s

File: libs/sdk-ui-kit/src/Timepicker/typings.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+// (C) 2020 GoodData Corporation
+
+/**
+ * @internal
+ */
+
+export type OverlayPositionType = "absolute" | "fixed" | "sameAsTarget";
```
Comment (no23reason) [inline libs/sdk-ui-kit/src/Timepicker/typings.ts:7]: I think this should not be defined here as it is related to Overlay, not TimePicker. We should probably move it to root typings for now (until Overlay is migrated here as well).

Comment (no23reason) [reply libs/sdk-ui-kit/src/Timepicker/typings.ts:7]: Or even better to typings/overlay.ts?

**Review by no23reason — COMMENTED** [review]
What is the purpose of the empty libs/sdk-ui-kit/styles/scss/Timepicker/_variables.scss?

File: libs/sdk-ui-kit/src/Timepicker/typings.ts (hunk 1 above)
Comment (NestorEncinas) [reply libs/sdk-ui-kit/src/Timepicker/typings.ts]: Moved to `typyings/overlay.ts`

**Comment by NestorEncinas** [comment]
Git didn;t get the delete folder change. Should not be there now

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1166/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/2772/ : SUCCESS in 11m 59s

**Comment by NestorEncinas** [comment]
extended test

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1167/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/402/ : UNSTABLE in 27m 33s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1174/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/2778/ : SUCCESS in 12m 15s

**Comment by NestorEncinas** [comment]
extended test

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1175/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/405/ : SUCCESS in 16m 44s

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/434/ : SUCCESS in 23s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/692/ : SUCCESS in 13s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/2779/ : SUCCESS in 12m 35s

**Pull request merged**